pkg/pillar/cmd/logmanager/domainstatus.go

```diff
@@ -51,6 +55,17 @@ func handleDomainStatusDelete(ctxArg interface{}, key string,
 	}
 	log.Infof("handleDomainStatusDomain remove %s",
 		status.DomainName)
-	delete(domainUuid, status.DomainName)
+	newName := nameRemLastdotNum(status.DomainName)
+	delete(domainUuid, newName)
 	log.Infof("handleDomainStatusDelete done for %s", key)
 }
+
+// XXX remove the .num from the domain name for now
+func nameRemLastdotNum(dName string) string {
+	dNames := strings.Split(dName, ".")
+	numPart := dNames[len(dNames)-1]
```

Comment:
Should we make this be robust against a future when there might not be a .<domainid> in the name?